feature/gribi/ate_tests/get_rpc_test/get_rpc_test.go

```diff
@@ -318,7 +322,7 @@ func testIPv4LeaderActive(ctx context.Context, t *testing.T, args *testArgs) {
 	// are returned, with no entry returned for 198.51.100.192/64.
 	dc := args.dut.Config()
 	ni := dc.NetworkInstance(*deviations.DefaultNetworkInstance).
-		Protocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "STATIC")
+		Protocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, *protocolName)
```

Comment:
Suggest making this `DEFAULT`, since that is the OC compliant default static protocol name.

If `DEFAULT` doesn't work, then add a `deviation`

Reply:
Hi @Ankur19, to the best of my knowledge  OC does not specify the name for static protocol name.  This value is vendor specific and can be anything. So, I do not think using values other than "DEFAULT" is an OC deviation.

Reply:
@mojiiba you PR makes the name configurable for different testbed (which I think it does make sense). [The OC](https://github.com/openconfig/public/blob/95948a506f12c904e7414e7d9cc27c467bee9070/release/models/network-instance/openconfig-network-instance.yang#L1271) does specify default value as `DEFAULT`, I"m not sure why the flag value can not be set to `DEFAULT`, since it's configurable anyway?

Reply:
@xw-g sorry for the confusion. Flag can be set to anything including DEFAULT. I meant value other than "DEFAULT" should not be considered as a deviation.